Add "restrictions on use" to the fork section (#36)
msagansk/Augur.Guide: collections/_version2-english/9-fork/3-restrictions-on-use.md

## During a Fork
"During a fork" means "during the [forking period]({{glossary_forking_period}})". During this period, REP holders cannot stake [REP]({{glossary_reputation_token}}) on any [market]({{glossary_market}}) in the [parent universe]({{glossary_parent_universe}}), since REP in the parent universe needs to be migrated to one of the [child universes]({{glossary_child_universe}}). Therefore, markets in the parent universe are never [finalized]({{glossary_finalized_market}}). 

Even if a market in the parent universe has not been migrated to the [winning universe]({{glossary_winning_universe}}), users may trade and [settle]({{glossary_settlement}}) their [shares]({{glossary_share}}) on the market. It does not matter whether the market is already finalized or not, users can do those on any market in the parent universe. Trading in the parent universe, traders pay [creator fee]({{glossary_creator_fee}}) and [reporting fee]({{glossary_reporting_fee}}), and those fees are added to [reporting fee pool]({{glossary_reporting_fee_pool}}). So users may purchase [participation tokens]({{glossary_participation_token}}) to collect those fees and redeem them.

> Users also may create a market in the parent universe. However, after a [fork]({{glossary_fork}}) started, a market in the parent universe never been finalized. To finalize a market, someone needs to migrate it to the winning universe (See [migration of the objects]({{url_migration_of_the_objects}}#non-forking-market) for details).

Reviewer: ```suggestion
Users may also create a market in the parent universe. However, after a [fork]({{glossary_fork}}) has started, a market in the parent universe can not be finalized. To finalize a market, someone needs to migrate it to the winning universe (See [migration of the objects]({{url_migration_of_the_objects}}#non-forking-market) for details).
```

> In child universes, there are no restrictions. It doesn't matter whether a child universe is winning one or losing one. You can do everything on them as soon as they are created. (Note: child universes are not created as soon as a fork starts. They are created when the first time REP is migrated to them. See [migration of the objects]({{url_migration_of_the_objects}}#forking-market) for details.)

Reviewer: ```suggestion
In child universes, there are no restrictions. It doesn't matter whether a child universe is winning one or losing one. You can do everything on it as soon as it is created.  Note: child universes are not created as soon as a fork starts, they are created the first time REP is migrated to them.  See [migration of the objects]({{url_migration_of_the_objects}}#forking-market) for details.)
```
